### PR TITLE
Feature/#1424/app service docs

### PR DESCRIPTION
**Pull request by mclacore**
App service doc and assets for users to use as a guide when building their own Azure app service. 

I used the `getting started` and `deploy` pages as a template and modified for Azure.

I also updated `getting started` and `deploy` for grammar, UI updates, and other minor enhancements.

### CODE REVIEW

File: docs/applications/02-getting-started.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Before getting started you'll need:
 
 :::
 
-In this guide we'll generate an `application bundle` and **publish** it to Massdriver. The application bundle will be all of the configuration your application needs to connect to runtimes (serverless, containers) and cloud resources (queues, databases, etc). Once the application bundle is published, we'll configure it in the UI and set up continious deployments from GitHub actions.
+In this guide we'll generate an `application bundle` and **publish** it to Massdriver. The application bundle will be all of the configuration your application needs to connect to runtimes (serverless, containers) and cloud resources (queues, databases, etc). Once the application bundle is published, we'll configure it in the UI and set up continuous deployments from GitHub actions.
```
Comment (WillBeebe) [inline docs/applications/02-getting-started.md:18]: thank you

File: docs/applications/02-getting-started.md [hunk 2]
```diff
@@ -156,18 +156,18 @@ For this walkthrough our application doesn't have any requirements that need to
 
 A breakdown of the fields:
 
-* .connections - all connected infrastructure and applications.
-* .connections.sqs - the connected SQS queue (if required & present).
-* .connections.sqs.data - sensitive information in the SQS artifact.
-* .connections.sqs.data.security - security related information for SQS.
-* .connections.sqs.data.security.iam - Principal of least privilege IAM policies exposed by this SQS bundle.
-* .connections.sqs.data.security.iam.subscribe - The `subscribe` policy your application needs.
+* `.connections` - all connected infrastructure and applications.
```
Comment (WillBeebe) [inline docs/applications/02-getting-started.md:159]: I like it

File: docs/applications/03-deploying-application.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-id: applications-deploying
-slug: /applications/deploying
+id: applications-deploying-application
+slug: /applications/deploying-application
```
Comment (WillBeebe) [inline docs/applications/03-deploying-application.md:3]: Good good

File: docs/applications/03-deploying-application.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ You'll need to add:
 * massdriver/aws-eks-cluster
 * massdriver/aws-rds-postgres or massdriver/aws-aurora-serverless-postgres
 
-After adding each to the canvas click "Configure." Feel free to fine tune the configuration, but if you are new to cloud infrastructure the "Configuration Presets" are a great way to get started quickly. Select a preset like "Development" and then click "Save & Deploy."
+After adding each to the canvas click **Configure**. Feel free to fine tune the configuration, but if you are new to cloud infrastructure the `Configuration Presets` are a great way to get started quickly. Select a preset like **Development** and then click **Deploy**.
```
Comment (WillBeebe) [inline docs/applications/03-deploying-application.md:60]: Observant points ++.  "Save and Deploy" => "Deploy" 💯

File: docs/applications/build-your-own/02-azure-app-service.md [hunk 5]
```diff
@@ -0,0 +1,288 @@
+---
+id: applications-build-your-own-azure
+slug: /applications/build-your-own/azure-app-service
+title: Azure App Service
+sidebar_label: Azure App Service
+---
+# Table of contents
```
Comment (WillBeebe) [inline docs/applications/build-your-own/02-azure-app-service.md:7]: Overal, I really like the organization of this. And how you link to other parts of our docs 👌

File: docs/applications/build-your-own/02-azure-app-service.md [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+
+First, create a new application bundle:
+
+```shell
+mass app new
+```
+
+Enter your application name and a description. These will be used in the Massdriver UI to identify application bundles to users.
+
+:::note
```
Comment (WillBeebe) [inline docs/applications/build-your-own/02-azure-app-service.md:34]: Great use of `note`

File: docs/applications/build-your-own/02-azure-app-service.md [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+The `massdriver.yaml`is covered in detail under [Bundle Configuration](/bundles/configuration) section.
+
+Its important to note that the generated code is a great way to _get started_, but you can customize the fields in `massdriver.yaml` and the rigging code in `src` to meet your needs.
+
+:::
+
+
+In addition to the configuration options exposed in the _bundle guide_, two additional configuration options are exposed to application bundles:
+
+* `app.envs` - _Map_ of environment variables to set on the running application.
```
Comment (WillBeebe) [inline docs/applications/build-your-own/02-azure-app-service.md:99]: If might be good to put an example app block here, thoughts?

```
app:
  envs:
    EXAMPLE_HOST: ....
  policies: []
```

File: docs/applications/build-your-own/02-azure-app-service.md [hunk 8]
```diff
@@ -0,0 +1,288 @@
...
+#### Environment Variable Examples
+
+Environment variables can be extracted from `parameters` and/or `connections`. Massdriver parses `jq` expressions to allow for rich composition of values into environment variables:
+
+The following would set the `LOG_LEVEL` in your application based on the value a users selects from the `log_level` dropdown.
+
+The `.params` prefix tells massdriver where to pull the value from. The field names can be any complex expression with JQ as we'll see in the next example.
+
+```yaml title="Simple param-based environment variable"
+app:
```
Comment (WillBeebe) [inline docs/applications/build-your-own/02-azure-app-service.md:113]: Ahh :). Nevermind.

File: docs/applications/build-your-own/02-azure-app-service.md [hunk 9]
```diff
@@ -0,0 +1,288 @@
...
+      $ref: massdriver/azure-virtual-network
+    azure_authentication:
+      $ref: massdriver/azure-service-principal
+    postgres:
+      $ref: massdriver/postgresql-authentication
+```
+
+## Publish
+
+At this point the application bundle can be published to the Massdriver bundle manager:
```
Comment (WillBeebe) [inline docs/applications/build-your-own/02-azure-app-service.md:196]:
```suggestion
At this point the application bundle can be published to the Massdriver package manager:
```

File: docs/applications/build-your-own/02-azure-app-service.md [hunk 10]
```diff
@@ -0,0 +1,288 @@
...
+
+```yaml
+mass app publish
+```
+
+You should see a success message.
+
+If you would like to automatically publish your application bundle in CI/CD, see the [GitHub Actions guide](/bundles/github-action).
```
Comment (WillBeebe) [inline docs/applications/build-your-own/02-azure-app-service.md:204]: 💯

**Review by WillBeebe — APPROVED** [review]
@mclacore this is really solid, I love it. I think it adds a ton of value for our Azure customers and raises the quality of the docs overall with your other images and edits. 🙌

File: docs/applications/build-your-own/02-azure-app-service.md (hunk 9 above)
Comment (mclacore) [reply docs/applications/build-your-own/02-azure-app-service.md:196]: Is it the `package manager`? I copy/pasta'd from the other section written by @coryodaniel

Comment (xpositivityx) [reply docs/applications/build-your-own/02-azure-app-service.md:196]: the software which manages the storage of custom bundles is called a package manager. Think of it like npm in javascript or apt for linux.

**Review by xpositivityx — APPROVED** [review]

**Pull request merged**